### PR TITLE
Making one of the static method public

### PR DESCRIPTION
**Pull request by drudenko**

### CODE REVIEW

**Comment by drudenko** [comment]
ping @haozhun @cberner

File: src/main/java/io/airlift/slice/SliceUtf8.java [hunk 1]
```diff
@@ -428,7 +428,7 @@ public static Slice fixInvalidUtf8(Slice slice, OptionalInt replacementCodePoint
         return utf8.slice(0, utf8Position);
     }
 
-    private static int getCodePointAtSafe(Slice utf8, int position)
+    public static int getCodePointAtSafe(Slice utf8, int position)
```
Comment (dain) [inline src/main/java/io/airlift/slice/SliceUtf8.java]: Lets rename this to `tryGetCodePointAt`.  Also, this method will need java docs explaining the return values (the negative number and what they mean).

**Comment by drudenko** [comment]
Applied changes, requested by @dain

**Comment by dain** [comment]
Merged, thanks!

**Comment by drudenko** [comment]
Thanks

**Pull request closed without merging**